config/peerpods/podvm/osc-podvm-delete-job.yaml

```diff
@@ -27,6 +27,8 @@ spec:
               value: "" # Set this to the aws ami id to delete
             - name: IMAGE_ID
               value: "" # Set this to the azure image id to delete
+            - name: IMAGE_ID
+              value: "" # Set this to the libvirt image id to delete
```

Comment:
Maybe LIBVIRT_VOL_NAME? because this is what will actually be deleted and to differ from Azure? (or LIBVIRT_IMAGE_ID to support future usage as you mentioned)

Reply:
Better to have LIBVIRT_IMAGE_ID now to avoid confusion of image and volume name.